src/components/containers/Cards/Items/PlaylistVideoTimeCard.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
+import { PlaylistVideoTimeCardProps } from "@/interfaces"; // ✅ Named import
+
+const PlaylistVideoTimeCard = ({ usertime = 0 }: PlaylistVideoTimeCardProps) => {
+  const [time, setTime] = useState(usertime * 60); // Convert minutes to seconds
+  const [isRunning, setIsRunning] = useState(false);
+
+  useEffect(() => {
+    setTime(usertime * 60);
+  }, [usertime]);
+
+  useEffect(() => {
+    let timer: NodeJS.Timeout | undefined;
```

Comment:
why use Node.js at all. We should keep it simple. In UI code we can't use this syntax